src/python/WMCore/ReqMgr/DataStructs/DefaultConfig/PERMISSION_BY_REQUEST_TYPE.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function, division
+
+ALL_ROLES = ['developer', 'admin', 'data-manager', 'production-operator', 'web-service']
+ALL_GROUP = ['reqmgr', 'facops', 'dataops']
+ADMIN_ROLES = ['admin', 'web-service']
+ADMIN_GROUP = ['reqmgr', 'facops']
+
+ADMIN_PERMISSION = {'role': ADMIN_ROLES, 'group': ADMIN_GROUP}
+DEFAULT_STATUS_PERMISSION = {'role': ALL_ROLES, 'group': ALL_GROUP}
+
+
+PERMISSION_BY_REQUEST_TYPE = {
+    "ADMIIN" : ADMIN_PERMISSION,
+    "NOSTATUS":  ADMIN_PERMISSION, # permmission for updating the values without updating status
```

Comment:
Typo in "ADMIIN". This NOSTATUS involves job splitting and priority, so Ops should be able to update it as well.